_posts/2020-03-16-dask-usage-at-blue-yonder.markdown

```diff
@@ -0,0 +1,161 @@
+---
+layout: single
+title:  "Dask Usage at Blue Yonder"
+date:   2020-03-16 10:00:00 +0100
+tags: technology python data-engineering
+header:
+  overlay_image: assets/images/tech_gear_banner.jpg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Andreas Merkel, Kshitij Mathur
+author_profile: true
+---
+# Dask Usage at Blue Yonder
+
+Back in 2017, Blue Yonder started to look into 
+[Dask/Distributed](https://distributed.dask.org) and how we can leverage it to power
+our machine learning and data pipelines.
+It's 2020 now, and we are using it heavily in production for providing demand predictions,
+recommended order quantities, and price proposals to our customers.
```

Comment:
Not sure if an external reader actually understands what we mean with demand predictions and oder quantiles. I think we should assume that a reader does not actually know what Blue Yonder does (and even if smbdy knows BY, BY is nowadays a quite large company with many applications)